app/assets/javascripts/backbone/views/loan_questionnaires_view.coffee

```diff
@@ -5,37 +5,34 @@ class MS.Views.LoanQuestionnairesView extends Backbone.View
   initialize: (options) ->
     @loanId = options.loanId
     @refreshContent()
+    @filterSwitchView = new MS.Views.FilterSwitchView()
 
   events:
     'ajax:error': 'submitError'
-    'click .filter-switch .btn': 'changeQuestionnaire'
 
   refreshContent: ->
     MS.loadingIndicator.show()
-    @$('.questionnaires-content').empty()
-    $.get "/admin/loans/#{@loanId}/questionnaires", (html) =>
+    @$('.questionnaires-content').load "/admin/loans/#{@loanId}/questionnaires", =>
       MS.loadingIndicator.hide()
-      @$('.questionnaires-content').html(html)
-      @initTopButtons()
+      @initializeTree()
+      @filterSwitchView.filterInit()
 
   submitError: (e) ->
     e.stopPropagation()
     MS.errorModal.modal('show')
     MS.loadingIndicator.hide()
 
-  initTopButtons: ->
-    selected = URI(window.location.href).query(true)['selected'] || 'criteria'
-    @showQuestionnaire(selected)
-    @$('.filter-switch .btn').removeClass('active')
-    @$(".filter-switch .btn[data-attrib=#{selected}]").addClass('active')
-
-  changeQuestionnaire: (e) ->
-    selected = $(e.currentTarget).closest('.btn').data('attrib')
-    @showQuestionnaire(selected)
-    url = URI(window.location.href).setQuery('selected', selected).href()
-    history.pushState(null, "", url)
-
-  showQuestionnaire: (attrib) ->
-    @$('.questionnaire').hide()
-    @$(".questionnaire[data-attrib=#{attrib}]").show()
-
+  initializeTree: ->
+    tree = @$('.jqtree')
+    tree.tree
+      dragAndDrop: false
+      selectable: false
+      useContextMenu: false
+      onCreateLi: (node, $li) =>
+        $li.attr('data-id', node.id)
+            .find('.jqtree-element')
+            .append(@$(".question[data-id=#{node.id}] > .explanation"))
+            .append(@$(".question[data-id=#{node.id}] > .answer-wrapper"))
+
+    tree.each ->
+      $(this).tree 'loadData', $(this).data('data')
```

Comment:
I think some quick comments on the above `onCreateLi` and `tree.each` are in order. It is not clear what is going on.